src/os_view.eliom

```diff
@@ -3,10 +3,25 @@
    and integrate it in your app to customize the behaviour according to
    your needs. *)
 
+(** This module defines functions to create password forms, connection forms,
+ * settings buttons and other stuff common contents to easily create content
+ * and start your application. Feel free to add other contents you need.
+ * As Eliom_content.Html.F is opened by default, if the module D is not
+ * explicitly used, HTML tags will be functional.
+ *)
+
 [%%shared
   open Eliom_content.Html
   open Eliom_content.Html.F
 
+(* generic_email_form ~a ~label ~text ~service () creates an email POST form
+ * with an input of type email and a submit button.
+ * Parameters:
+ * ~a: modify the attributes of the form.
+ * ~label: add a label (none by default) to the email input.
+ * ~text: text for the button ["Send" by default].
+ * ~service: service which data are sent to.
```

Comment:
"service which the data is sent to"? Treating data as a mass noun sounds way more correct (to me).